### PR TITLE
All the changes

### PR DESCRIPTION
**Pull request by bsipocz**
**NOT TO BE MERGED!**

I open this PR to have all the changes in one place to be able to leave comments for particular lines.

I may be able to do some git trickery to only trck the tex files here, but anyway, will leave this here for now.

### CODE REVIEW

**Comment by keflavich** [comment]
Right, so this is a 'never-merge' PR.

**Comment by bsipocz** [comment]
yes. I've made it as I have tons of comments for the current draft, but many are really a comment so I can't put that in a PR. 

However, if you prefer another way to do it, please let me know.

**Comment by eteq** [comment]
A thought on (currently) section 2 ("The software"): It currently reads as sort of a "developer guide" rather than a "what's in the software".  @keflavich and I discussed and I think I agree that it makes sense to implement #32 to help address this.

Maybe also the intro to that section can begin with "this is what astroquery does for the user - queries catalogs, images, etc." Then the current intro can be moved later as a new sub-section ("How modules are structured", maybe?), and the current 2.3 ("The API") could be moved to be 2.1?

File: main.tex [hunk 1]
```diff
@@ -0,0 +1,423 @@
...
+
+\astroquery also receives contributions from funded programs. For instance, the
+JPLHorizons module has been implemented as part of the \texttt{sbpy}
+project\footnote{\url{http://sbpy.org}} with support from NASA PDART grant
+80NSSC18K0987. Further Solar System-related services are planned to be added to
+\astroquery through this support.
+
+
+\astroquery has received support from the Google Summer of Code
+program, with two students (co-authors Madhura Parikh and Simon Liedtke)
```
Comment (migueldvb) [inline main.tex]: Liedtke worked on SunPy through the GSoC?

Comment (migueldvb) [reply main.tex]: Maybe you mean Ayush Yadav

Comment (keflavich) [reply main.tex]: No, Ayush never worked on astroquery, he dropped out.  Simon did.  We picked him up the year after he worked on sunpy

**Comment by bsipocz** [comment]
Closing this as obsolete.

**Pull request closed without merging**